app/controllers/tag_set_nominations_controller.rb

```diff
@@ -1,22 +1,22 @@
 class TagSetNominationsController < ApplicationController
   cache_sweeper :tag_set_sweeper
-  
+
   before_filter :users_only
   before_filter :load_tag_set, :except => [ :index ]
   before_filter :check_pseud_ownership, :only => [:create, :update]
   before_filter :load_nomination, :only => [:show, :edit, :update, :destroy]
   before_filter :set_limit, :only => [:new, :edit, :show, :create, :update, :review]
 
   def check_pseud_ownership
-    if params[:tag_set_nomination][:pseud_id]
-      pseud = Pseud.find(params[:tag_set_nomination][:pseud_id])
+    if tag_set_nomination_params[:pseud_id]
```

Comment:
More specific error messages sound good, so let's use this one. Thanks for the explanation!